addons/web/controllers/main.py

```diff
@@ -1049,7 +1049,8 @@ def content_image(self, xmlid=None, model='ir.attachment', id=None, field='datas
         if status == 301 or (status != 200 and download):
             return request.env['ir.http']._response_by_status(status, headers, content)
         if not content:
-            content = base64.b64encode(self.placeholder(image='placeholder.png'))
+            placeholder = kw.get('placeholder', 'placeholder.png')
```

Comment:
why not put `placeholder='placeholder.png' in the method signature?`

Reply:
hello @alexkuhn , 
seems better.! :+1: 
can you please check it, I've done changes ?